variants/rak4631_nomadstar_meteor_pro/platformio.ini

```diff
@@ -0,0 +1,56 @@
+; NomadStar Meteor Pro based on RAK4631 with RGBW LED LP5562 support
+[env:rak4631_nomadstar_meteor_pro]
+extends = nrf52840_base
+board = wiscore_rak4631
+board_check = true
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak4631_nomadstar_meteor_pro -D NOMADSTAR_METEOR_PRO
+  -L "${platformio.libdeps_dir}/${this.__env__}/bsec2/src/cortex-m4/fpv4-sp-d16-hard"
+  ;-DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -DEINK_DISPLAY_MODEL=GxEPD2_213_BN
+  -DEINK_WIDTH=250
+  -DEINK_HEIGHT=122
+  -DRADIOLIB_EXCLUDE_SX128X=1
+  -DRADIOLIB_EXCLUDE_SX127X=1
+  -DRADIOLIB_EXCLUDE_LR11X0=1
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/rak4631_nomadstar_meteor_pro> +<mesh/eth/> +<mesh/api/> +<mqtt/>
+lib_deps = 
+  ${nrf52840_base.lib_deps}
+  ${networking_base.lib_deps}
+  melopero/Melopero RV3028@^1.1.0
```

Comment:
See associated comments for the variant.h file, please remove unneccesary libs from the linker.